uiowa/uiowa
# 2 plus 2 part 3

Resolves #5228 





# How to test










- `git checkout 2-plus-2-part-3 && git pull`
- `blt ds --site=admissions.uiowa.edu`
- Create some Community College taxonomy terms
- Create some Major (2 plus 2) content pieces connecting community colleges to areas of study
- Check out the new 'list of all available majors' page @ https://admissions.uiowa.ddev.site/2plus2/majors
   - Check that the following AoSs are usable by creating associated majors and checking their corresponding views pages
     - https://admissions.uiowa.ddev.site/academics/nursing-rn-bsn
     - https://admissions.uiowa.ddev.site/academics/resilience-and-trauma-informed-perspectives-certificate
     - https://admissions.uiowa.ddev.site/academics/interscholastic-athleticactivities-administration
- Check out the new 'list of all participating community colleges' page @ https://admissions.uiowa.ddev.site/2plus2/colleges
   - Try creating a community college with a '-' or '/' in the name, and check that the view and its linking still work
